### PR TITLE
shorter version of blog post

### PR DESCRIPTION
**Pull request by cobordism**
a radical shortening - may it serve as a space for discussion.

### CODE REVIEW

File: blog/poc3.md [hunk 1]
```diff
@@ -1,120 +1,64 @@
 
 
-Swarm reaches a significant milestone as the next generation of its proof of concept releases (POC3) is now deployed on the public Testnet. The foundation is running a 50-node strong cluster (with a public gateway on https://swarm-gateways.net) hopefully growing as our collaborators switch on their permanent nodes and clusters.
-The POC3 code is now merged into the official [go-ethereum repository's master branch](https://github.com/ethereum/go-ethereum).
-It has been a year and a half since the first release of the POC2 series was deployed and [the Swarm project launched its public alpha network](https://blog.ethereum.org/2016/12/15/swarm-alpha-public-pilot-basics-swarm/).
-Two [Swarm summits](https://swarm-gateways.net/bzz:/swarm-summit.eth), two [orange papers](http://swarm-guide.readthedocs.io/en/latest/resources.html#orange-papers) and [forty thousand lines of code](https://github.com/ethereum/go-ethereum/pull/17041) later, it is time to give an update.
+The Swarm Team is pleased to announce the immediate release of Swarm 0.3, the third proof-of-concept release (POC3) of the Ethereum Swarm client.
```
Comment (holisticode) [inline blog/poc3.md]: I wonder if we should use "client" for a p2p binary. "Node", "Binary" are other options, possibly there are better ones.

File: blog/poc3.md [hunk 2]
```diff
@@ -1,120 +1,64 @@
...
 
-## Basics of Swarm
+Swarm 0.3 has been deployed to the public Testnet, and the Ethereum foundation is running a 50-node strong cluster of Swarm nodes, together with a public web gateway on https://swarm-gateways.net.
```
Comment (holisticode) [inline blog/poc3.md]: Shouldn't Foundation be upper case?

File: blog/poc3.md [hunk 3]
```diff
@@ -1,120 +1,64 @@
...
-* mutability (ENS, MRU)
-* node-to-node messaging (PSS)
-* access control (ACT)
-* payment and service agreements, escrow and user protection (SW3)
-* database services (POT)
-* search (like Google, not discussed here)
+At its core, Swarm is a service that provides APIs to upload and download content to the crowd, and through a url-based addressing offers *virtual hosting of websites* and decentralised applications (dapps) without webservers, using decentralised peer-to-peer distributed infrastructure.  The vision is that of an new internet which is not only fault-tolerant, has zero downtime and offers censorship resistance but is also economically self-sustaining due to a built-in incentive system. By compensating nodes for contributing their bandwidth and disk space, these incentives aim to achieve reliable low-latency scalable retrieval of popular content on the one hand and guarantees persistence of important yet rarely accessed data like archives or backups on the other. For smooth delivery Swarm will use the SWAP protocol (planned for POC3.1) while for storage guarantees it will use a two-tiered insurance system (planned for POC4).
```
Comment (holisticode) [inline blog/poc3.md]: to the crowd? Sounds odd to me

File: blog/poc3.md [hunk 4]
```diff
@@ -1,120 +1,64 @@
...
-Using ENS for domain name resolution, the URL scheme provides
-content retrieval based on mnemonic (or branded) names, much like the DNS of the world wide web, but without servers.
-Mutable Resource Update is, on the other hand, an off-chain solution for communicating updates to a resource, it offers cheaper and potentially faster updates than ENS, yet the updates can be consolidated on ENS by any third party willing to pay for the transaction.
+Another experimental new feature in POC3 is the Swarm Mutable Resource.
+Typical in p2p storage systems, content as addressed by it digital fingerprint (hash) and any changes to the content radically change this address. Users of the web, however, are accustomed to mutable resources: when visiting URL-s we expect to see the most up to date version of the 'site'. In order to make it easy to access changing content at predictable addresses, Swarm integrates with the Ethereum Name Service [(ENS)](http://swarm-guide.readthedocs.io/en/latest/usage.html#Ethereum) on the Ethereum blockchain. This is what allows us to to reference swarm content by names like [(]bzz://theswarm.eth](https://swarm-gateways.net/bzz:/theswarm.eth/).
```
Comment (holisticode) [inline blog/poc3.md]: addressed by its

Comment (holisticode) [reply blog/poc3.md]: visiting URLs vs URL-s?

File: blog/poc3.md [hunk 5]
```diff
@@ -1,120 +1,64 @@
...
-Due to the fact that Swarm retrieval is on-demand, Swarm FUSE mounts are readable with no latency. Swarm FUSE supports file system write operations (creating, appending, updating files) by maintaining the collection's Swarm hash as an internal state.
-Combining FUSE with MRU, on each change we can issue a mutable resource update, this makes it possible to think of FUSE to sync your entire home folder between devices, virtually serving as the backend to a decentralised drop box.
+[FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to integrate Swarm data directly into their local file systems (only on Linux and Mac). Using this system, users can "mount a Swarm manifest" as if it were a regular directory. It supports file system read and write operations, in which all content is automatically synced with the Swarm.
+In future, combining FUSE with Swarm Mutable Resources, it should be possible sync your entire home folder between devices - the backend to a decentralised storage with dropbox-like functionality.
```
Comment (holisticode) [inline blog/poc3.md]: it should be possible **to** sync

Comment (holisticode) [reply blog/poc3.md]: your entire home folder or just "any folder"?

**Review by holisticode — APPROVED** [review]

File: blog/poc3.md [hunk 6]
```diff
@@ -1,120 +1,64 @@
...
```
Comment (janos) [inline blog/poc3.md]: dropbox -> Dropbox

File: blog/poc3.md [hunk 7]
```diff
@@ -1,120 +1,64 @@
...
 
-Swarm POC3 supports [*encryption*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Encryption) so protecting your private data has never been easier. Once *Access Control Trees* are implemented, we will have a fully fledged solution for 'sharing' a subdirectory, a feature well known from Dropbox or Google Drive. Combining with PSS the 'share' can be sent in a notification to the recipient.
 
 # The year ahead
 
-Just as the past year, these coming 10+ month will be both exciting and challenging. We are on target delivering Swarm POC4 (production beta prerelease) in the first half of 2019.
+The year ahead will be both exciting and challenging. As part of the POC3 series, we are planning to switch on a revamped SWAP accounting system (Swarm 0.3.1),
+Implementing access control and authentication for Swarm content, and enabling 'light' Swarm nodes  (Swarm 0.3.2).
```
Comment (janos) [inline blog/poc3.md]: Remove two spaces between `nodes  (Swarm 0.3.2)`

File: blog/poc3.md (hunk 4 above)
Comment (zelig) [inline blog/poc3.md]: by its

File: blog/poc3.md [hunk 8]
```diff
@@ -1,120 +1,64 @@
...
-
-With `pss` you can send messages to any node in the Swarm network.
-The messages are routed in the same manner as retrieve requests for chunks.
-Due to forwarding kademlia, pss offers sender anonymity.
-Since `pss` messages are encrypted (using whisper's crypto), ultimately *the recipient is whoever can decrypt the message*. Encryption can be done using asymmetric or symmetric encryption methods.
-Thanks to end-to-end encryption, pss caters for private communication.
+The same p2p connections that are used for data storage and delivery, can also be used for node-to-node messaging. PSS combines Swarm routing (the `bzz` protocol) with the Whisper (`shh`) encrypted message format (`bzz`+`shh`=`pss`). In short, PSS is a [messaging protocol](http://swarm-guide.readthedocs.io/en/latest/pss.html) with strong privacy features running on top of the Swarm network. This messaging infrastructure can be the foundation of a whole new system of internode communication services (the email, tweet, newsletter of the future).
```
Comment (zelig) [inline blog/poc3.md]: technically it is not the 'bzz PROTOCOL' actually, so just put (`bzz`) there

File: blog/poc3.md [hunk 9]
```diff
@@ -1,120 +1,64 @@
...
-The Swarm network implements a *distributed preimage archive (DPA)*, which is a  type of content addressed distributed chunk store in which nodes closest to the address of a chunk do not only serve information about the content but actually host the data. This DPA is the underlying distributed component of the Swarm service. When you upload content, the chunks need to reach the nodes that are closest to it (a process called *syncing*), so that when you retrieve chunks, nodes can send requests relayed towards the chunk's address via ever closer peers.
+It has been a year and a half since the first release of the POC2 series was deployed and [the Swarm project launched its public alpha network](https://blog.ethereum.org/2016/12/15/swarm-alpha-public-pilot-basics-swarm/).
+Two [Swarm summits](https://swarm-gateways.net/bzz:/swarm-orange-summit.eth), two [orange papers](http://swarm-guide.readthedocs.io/en/latest/resources.html#orange-papers) and [forty thousand lines of code](https://github.com/ethereum/go-ethereum/pull/17041) later, it is time to give an update.
```
Comment (zelig) [inline blog/poc3.md]: I know i rwote it but '[after a bloody long and eventful time] it is time to give an update' sounds like we were silent for too long...

File: blog/poc3.md [hunk 10]
```diff
@@ -1,120 +1,64 @@
...
-* mutability (ENS, MRU)
-* node-to-node messaging (PSS)
-* access control (ACT)
-* payment and service agreements, escrow and user protection (SW3)
-* database services (POT)
-* search (like Google, not discussed here)
+At its core, Swarm is a service that provides APIs to upload and download content to the crowd, and through url-based addressing offers *virtual hosting of websites* and decentralised applications (dapps) without webservers, using decentralised peer-to-peer distributed infrastructure.  The vision is that of an new internet which is not only fault-tolerant, has zero downtime and offers censorship resistance but is also economically self-sustaining due to a built-in incentive system. By compensating nodes for contributing their bandwidth and disk space, these incentives aim to achieve reliable low-latency scalable retrieval of popular content on the one hand and guarantees persistence of important yet rarely accessed data like archives or backups on the other. For smooth delivery Swarm will use the SWAP protocol (planned for POC3.1) while for storage guarantees it will use a two-tiered insurance system (planned for POC4).
```
Comment (zelig) [inline blog/poc3.md]: URL-based (?)

File: blog/poc3.md (hunk 5 above)
Comment (zelig) [reply blog/poc3.md]: yes its better to say 

it should be possible to sync, for instance, your home folder between ...

File: blog/poc3.md [hunk 11]
```diff
@@ -1,120 +1,64 @@
...
 
-Swarm POC3 supports [*encryption*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Encryption) so protecting your private data has never been easier. Once *Access Control Trees* are implemented, we will have a fully fledged solution for 'sharing' a subdirectory, a feature well known from Dropbox or Google Drive. Combining with PSS the 'share' can be sent in a notification to the recipient.
 
 # The year ahead
 
-Just as the past year, these coming 10+ month will be both exciting and challenging. We are on target delivering Swarm POC4 (production beta prerelease) in the first half of 2019.
+The year ahead will be both exciting and challenging. As part of the POC3 series, we are planning to switch on a revamped SWAP accounting system (Swarm 0.3.1),
+Implementing access control and authentication for Swarm content, and enabling 'light' Swarm nodes  (Swarm 0.3.2).
+
+We are on target delivering Swarm POC4 (production beta prerelease) in 2019.
```
Comment (zelig) [inline blog/poc3.md]: :)

File: blog/poc3.md [hunk 12]
```diff
@@ -1,120 +1,64 @@
...
-Due to the fact that Swarm retrieval is on-demand, Swarm FUSE mounts are readable with no latency. Swarm FUSE supports file system write operations (creating, appending, updating files) by maintaining the collection's Swarm hash as an internal state.
-Combining FUSE with MRU, on each change we can issue a mutable resource update, this makes it possible to think of FUSE to sync your entire home folder between devices, virtually serving as the backend to a decentralised drop box.
+[FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to integrate Swarm data directly into their local file systems (only on Linux and Mac). Using this system, users can "mount a Swarm manifest" as if it were a regular directory. It supports file system read and write operations, in which all content is automatically synced with the Swarm.
+In future, combining FUSE with Swarm Mutable Resources, it should be possible sync your entire home folder between devices - the backend to a decentralised storage with dropbox-like functionality.
```
Comment (zelig) [inline blog/poc3.md]: we cannot possibly not mention encryption that i insist on :)

**Review by zelig — CHANGES_REQUESTED** [review]

File: blog/poc3.md (hunk 1 above)
Comment (cobordism) [reply blog/poc3.md]: Swarm client v0.3 ?

File: blog/poc3.md (hunk 9 above)
Comment (cobordism) [reply blog/poc3.md]: it is time to give an update -> it is time to take stock?

File: blog/poc3.md [hunk 13]
```diff
@@ -1,37 +1,35 @@
...
-
-
+POC3 code was finalised just in time for the [Swarm Orange Summit in Ljubljana](https://swarm-gateways.net/bzz:/swarm-orange-summit.eth), where we had 80 participants and a very inspiring and creative week ([watch this two-minute video hosted on Swarm](https://swarm-gateways.net/bzz:/079b4f4155d7e8b5ee76e8dd4e1a6a69c5b483d499654f03d0b3c588571d6be9/)) of talks and coding. It is inspiring to see a growing number of [contributors](https://github.com/ethersphere/go-ethereum/blob/b14d635539a7fd548bd1fe4fe987f137229ff38e/swarm/AUTHORS) and [companies that want to build on swarm](https://swarm-gateways.net/bzz:/swarm-orange-summit.eth/#mu-sponsors)
```
Comment (zelig) [inline blog/poc3.md]: . period at the end of the paragraph missing

File: blog/poc3.md [hunk 14]
```diff
@@ -1,37 +1,35 @@
...
+At its core, Swarm is a service that provides APIs to upload and download content to the crowd, and through URL-based addressing offers *virtual hosting of websites* and decentralised applications (dapps) without webservers, using decentralised peer-to-peer distributed infrastructure.  The vision is that of an new internet which is not only fault-tolerant, has zero downtime and offers censorship resistance but is also economically self-sustaining due to a built-in incentive system. By compensating nodes for contributing their bandwidth and disk space, these incentives aim to achieve reliable low-latency scalable retrieval of popular content on the one hand and guarantees persistence of important yet rarely accessed data like archives or backups on the other. For smooth delivery Swarm will use the SWAP protocol (planned for POC3.1) while for storage guarantees it will use a two-tiered insurance system (planned for POC4).
```
Comment (zelig) [inline blog/poc3.md]: to the crowd? hahaa

File: blog/poc3.md [hunk 15]
```diff
@@ -47,13 +45,16 @@ MRU is an experimental feature in current POC3 testnet and is still undergoing c
 ## FUSE support
 
 [FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to integrate Swarm data directly into their local file systems (only on Linux and Mac). Using this system, users can "mount a Swarm manifest" as if it were a regular directory. It supports file system read and write operations, in which all content is automatically synced with the Swarm.
```
Comment (zelig) [inline blog/poc3.md]: no comma after FUSE, enables

File: blog/poc3.md [hunk 16]
```diff
@@ -47,13 +45,16 @@ MRU is an experimental feature in current POC3 testnet and is still undergoing c
...
-In future, combining FUSE with Swarm Mutable Resources, it should be possible sync your entire home folder between devices - the backend to a decentralised storage with dropbox-like functionality.
+In future, combining FUSE with Swarm Mutable Resources, it should be possible, for example, to sync your entire home folder between devices - the backend to a decentralised storage with Dropbox-like functionality.
 
+## Encryption support
+
+Swarm 0.3 comes with built-in [*encryption*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Encryption) allowing for secure uploads of private data. With the planned implementation of *Access Control Trees*, we will have the ability to upload a directory privately and still 'share' a subdirectory with specific peers.
```
Comment (zelig) [inline blog/poc3.md]: well that is true but that is already possible. so I suggest:

allowing for secure uploads of private data. The way encryption works users can pload a directory privately and still 'share' a subdirectory with specific peers. 
*Access Control Trees* (Swarm 0.3.2) will offer an API for users to manage access to assets independantly of publishing content. Granted access will work across versions of a mutable resources.

**Review by zelig — CHANGES_REQUESTED** [review]

**Pull request closed without merging**